external/builder/babel-plugin-pdfjs-preprocessor.mjs

```diff
@@ -173,6 +192,19 @@ function babelPluginPDFJSPreprocessor(babel, ctx) {
           path.replaceWith(t.importExpression(source));
         }
       },
+      MemberExpression(path) {
+        // The Emscripten Compiler (emcc) generates code that allows the caller
+        // to provide the Wasm module (thorugh Module.instantiateWasm), with
+        // a fallback in case .instantiateWasm is not provided.
+        // We always define instantiateWasm, so we can hard-code the check
+        // and let our dead code elimination logic remove the unused fallback.
+        if (
+          path.parentPath.isIfStatement({ test: path.node }) &&
+          path.matchesPattern("Module.instantiateWasm")
```

Comment:
Would it be possible to add test(s) for this part in particular (in addition to the new ones already in the patch), such that we'll be "warned" if in the a future emcc changes the output format and thus breaks the dead code elimination?